### PR TITLE
[web] lock Chromium CIPD package version to Chrome 96

### PR DESCRIPTION
**Pull request by yjbanov**
Update test Chromium version to the CIPD package corresponding to Chromium 96. Add Chromium versions in some dependencies that didn't specify it.

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ platform_properties:
       dependencies: >-
         [
           {"dependency": "android_sdk", "version": "version:29.0"},
-          {"dependency": "chrome_and_driver", "version": "version:84"},
+          {"dependency": "chrome_and_driver", "version": "kkOzlCHhXTi3TpU7miUzqSOCVa_2RJVb09kfze0a6z0C"},
```
Comment (CaseyHillers) [inline .ci.yaml:87]: Can we tag this cipd ref as version:96 to make it easier to audit?

Comment (yjbanov) [reply .ci.yaml:87]: Do you have the command line to do this?

Comment (CaseyHillers) [reply .ci.yaml:87]: `cipd set-tag flutter_internal/browsers/chrome/linux-amd64 -tag version:96 -version kkOzlCHhXTi3TpU7miUzqSOCVa_2RJVb09kfze0a6z0C`

File: .ci.yaml [hunk 2]
```diff
@@ -84,7 +84,7 @@ platform_properties:
       dependencies: >-
         [
           {"dependency": "android_sdk", "version": "version:29.0"},
-          {"dependency": "chrome_and_driver", "version": "kkOzlCHhXTi3TpU7miUzqSOCVa_2RJVb09kfze0a6z0C"},
+          {"dependency": "chrome_and_driver", "version": "latest-96"},
```
Comment (CaseyHillers) [inline .ci.yaml:87]: In this case, what does latest indicate? Are there multiple releases of 96?

Comment (yjbanov) [reply .ci.yaml:87]: I'm not sure. I'm reusing the one we already have: https://chrome-infra-packages.appspot.com/p/flutter_internal/browsers/chrome/linux-amd64/+/

Comment (yjbanov) [reply .ci.yaml:87]: I'm finding out from @hterkelsen if I should maybe use `927388` instead.

Comment (CaseyHillers) [reply .ci.yaml:87]: There's also https://chrome-infra-packages.appspot.com/p/flutter_internal/browsers/chrome/linux-amd64/+/version:96

Comment (keyonghan) [reply .ci.yaml]: I think `verson: 96` is the way to go. We have been making other cipd packages following this consistent format, like android version: https://chrome-infra-packages.appspot.com/p/flutter_internal/android/sdk/linux-amd64/+/56xIdLBDFMMlbbbtLk_9czg8NB-4GjFIbxf5h3-BixQC
/cc @godofredoc

Comment (godofredoc) [reply .ci.yaml]: That's correct, we are moving towards pinning explicit versions in the ci.yaml file.

**Comment by skia-gold** [comment]
Gold has detected about 10 new digest(s) on patchset 2.
View them at https://flutter-gold.skia.org/cl/github/93289

**Comment by skia-gold** [comment]
Gold has detected about 7 new digest(s) on patchset 3.
View them at https://flutter-gold.skia.org/cl/github/93289

**Comment by yjbanov** [comment]
This is a change that will enable some progress on this issue: https://github.com/flutter/packages/pull/518

**Pull request closed without merging**